config/templates/postgresql-db.yaml

```diff
@@ -74,11 +74,11 @@ spec:
           synchronous_mode_strict: false
           ttl: 30
         podAnnotations:
-          com.flanksource.infra.logs/processors.0.drop_event.when.contains.message: "no action.  i am a secondary and i am following a leader"
-          com.flanksource.infra.logs/processors.1.drop_event.when.contains.message: "no action.  i am the leader with the lock"
-          com.flanksource.infra.logs/processors.2.drop_event.when.contains.message: "INFO: Lock owner:"
-          com.flanksource.infra.logs/processors.3.drop_event.when.contains.message: "INFO: does not have lock"
-          com.flanksource.infra.logs/processors.4.dissect.tokenizer: "%{date} %{time} %{TZ} [%{PID}] %{user} %{level}: %{log}"
+          com.flanksource.infra.logs/processors.0.drop_event.when.contains.message: no action.  i am a secondary and i am following a leader
+          com.flanksource.infra.logs/processors.1.drop_event.when.contains.message: no action.  i am the leader with the lock
+          com.flanksource.infra.logs/processors.2.drop_event.when.contains.message: 'INFO: Lock owner:'
+          com.flanksource.infra.logs/processors.3.drop_event.when.contains.message: 'INFO: does not have lock'
+          com.flanksource.infra.logs/processors.4.dissect.tokenizer: '%{date} %{time} %{TZ} [%{PID}] %{user} %{level}: %{log}'
```

Comment:
what was the motivation for changing these lines?

Reply:
I just run `make` at the root of the directory to re-generate the CRDs from go code. But it also ran lint and the linter made these changes.